src/Stratis.Bitcoin/Consensus/ConsensusManagerBehavior.cs

```diff
@@ -560,8 +572,11 @@ public void UpdateBestSentHeader(ChainedHeader header)
         }
 
         /// <summary>Tries to sync the chain with the peer by sending it <see cref="GetHeadersPayload"/> in case peer's state is <see cref="NetworkPeerState.HandShaked"/>.</summary>
-        public async Task ResyncAsync()
+        public void RequestHeaders()
```

Comment:
Was there no `await` in this call? I also dont see a return `Task` hmm very odd

Reply:
I'm going to revert this for now. We can clean this up later.